src/seedu/addressbook/ui/TextUi.java

```diff
@@ -114,7 +83,7 @@ public void showInitFailedMessage() {
     }
 
     /** Shows message(s) to the user */
-    public void showToUser(String... message) {
+    public static void showToUser(String... message) {
```

Comment:
In the code below, `LINE_PREFIX` + m.replace(...)` can also go into the `Support` class as it does formatting, no?